src/_layout/components/Layout/Layout.tsx

```diff
@@ -10,9 +11,11 @@ interface Props {
 export const Layout: FC<Props> = ({ children }) => {
   useScrollToTop();
 
+  const { width } = useWindowSize();
+
   return (
     <>
-      <NavbarDesktop />
+      {width > 768 ? <NavbarDesktop /> : <NavBarMobile />}
```

Comment:
This is a bit ugly, considering the fact that we use elsewhere the material theme to get the media query this is a point that should be modified manually, if we need to change the breakpoint. I would find a way to use the media query in the theme.